x/lockup/types/msgs_test.go

```diff
@@ -95,10 +89,217 @@ func TestMsgLockTokens(t *testing.T) {
 	}
 }
 
-// TODO: Complete table driven tests for the remaining messages
+func TestMsgBeginUnlockingAll(t *testing.T) {
+	appParams.SetAddressPrefixes()
+	addr1, invalidAddr := generateTestAddrs()
+
+	createMsg := func(after func(msg MsgBeginUnlockingAll) MsgBeginUnlockingAll) MsgBeginUnlockingAll {
+		properMsg := MsgBeginUnlockingAll{
+			Owner: addr1,
+		}
+
+		return after(properMsg)
+	}
+
+	msg := createMsg(func(msg MsgBeginUnlockingAll) MsgBeginUnlockingAll {
+		// Do nothing
+		return msg
+	})
+
+	require.Equal(t, msg.Route(), RouterKey)
+	require.Equal(t, msg.Type(), "begin_unlocking_all")
+	signers := msg.GetSigners()
+	require.Equal(t, len(signers), 1)
+	require.Equal(t, signers[0].String(), addr1)
```

Comment:
Can we please instead make the assertions inside the for loop for each test case?

Also, let's try to avoid the complexity with `createMsg` callbacks, please. I think we can directly initialize:
```
msg: MsgBeginUnlockingAll{
			Owner: addr1,
}
```
instead of having the callback overhead

Reply:
Yes thank you. I think it makes the code look better.